.buildkite/bk.fips-integration.pipeline.yml

```diff
@@ -0,0 +1,225 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
+  ASDF_MAGE_VERSION: 1.14.0
+  CUSTOM_IMAGE_TAG: "git-${BUILDKITE_COMMIT:0:12}"
+  CI_ELASTIC_AGENT_DOCKER_IMAGE: "docker.elastic.co/beats-ci/elastic-agent-fips-cloud"
+
+  IMAGE_UBUNTU_2404_X86_64: "platform-ingest-elastic-agent-ubuntu-2404-1744855248"
+  IMAGE_UBUNTU_X86_64_FIPS: "platform-ingest-fleet-server-ubuntu-2204-fips"
+  IMAGE_UBUNTU_ARM64_FIPS: "test-platform-ingest-fleet-server-ubuntu-2204-fips-aarch64-1747830486"
+
+steps:
+  - label: Build and push custom elastic-agent image
+    key: integration-fips-cloud-image
+    env:
+      ASDF_TERRAFORM_VERSION: 1.9.2
+      FIPS: "true"
+    command: |
+      #!/usr/bin/env bash
+      set -euo pipefail
+      mage cloud:image
```

Comment:
`cloud:image` invokes the [Package()](https://github.com/elastic/elastic-agent/blob/main/magefile.go#L982) function that packages the agent again. This step takes a considerable amount of time. Can we download the artifacts produced by the packaging step and reuse them?

Reply:
We can try, but I'm not sure if the Image step will use artifacts that are already present, @pkoutsovasilis do you know if it will?